ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml

```diff
@@ -404,7 +404,6 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-cgroupsv2
 - as: e2e-aws-crun
-  optional: true
```

Comment:
Should we explicitly mark it required by setting `optional: false`? Maybe this we don't rely on default value to be always `true`. If for some reason they decide to change the default behaviour of that `optional` field, it won't affect this job if we have it `optional: false` explicitly set. WDYT?

Reply:
good point... fixed

Reply:
I'll have to revert... the change fails on ci-operator-config-metadata's lint.